tools/importer-rest-api-specs/internal/components/terraform/models/work_in_progress_resource.go

```diff
@@ -0,0 +1,26 @@
+package models
+
+import (
+	"github.com/hashicorp/pandora/tools/data-api-sdk/v1/models"
+	"github.com/hashicorp/pandora/tools/sdk/config/definitions"
+)
+
+// WorkInProgressResource is an intermediate type used by the `terraform` package to enable
+// building a Terraform Resource component-by-component. It contains both the WIP Resource,
+// the InputData used to identify the resource (which can in turn contain test data and defaults)
+// but is ultimately only used within this package to build Resources.
+//
+// NOTE: this should only be used within the `terraform` package - with the SDK Model `TerraformResourceDefinition`
+// used elsewhere.
+type WorkInProgressResource struct {
```

Comment:
Yeah I wondered the same here tbh, I ended up deciding against it because the paths felt too long? Particularly when bringing in the API Definitions Parser, which is a bunch of nested directories, but perhaps that's something we fix in a follow-up when the Parser logic is refactored too?